src/services/completions.ts

```diff
@@ -1066,7 +1065,7 @@ function getExhaustiveCaseSnippets(
         }
 
         const newClauses = map(elements, element => factory.createCaseClause(element, []));
-        const newLineChar = getNewLineCharacter(options, maybeBind(host, host.getNewLine));
+        const newLineChar = getNewLineCharacter(options);

```

Comment:
Just to double check, this won't cause our completion snippets to have the wrong newlines, right? This is sort of awkward because it's the input newline character we'd want in snippets, not the output type. I'm not sure if `host.getNewLine` here was overriding that choice or what, but I guess there's no way this code could have determined the "right" thing to use without using the SourceFile, so, I'm confused.

Reply:
Hm, the current errors in baselines seem to imply that it does matter?